File: Company Profile/CompanyProfileWillyWarman/app/src/main/java/com/companyprofile/willywarman/layout/MainActivity.java
package com.companyprofile.willywarman.layout;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import com.companyprofile.willywarman.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        new Handler().postDelayed(() -> {
            Intent intent = new Intent(getApplicationContext(), DashboardActivity.class);
            startActivity(intent);
            finish();
        }, 2000);
    }
}

File: GKEBanjarbaru/app/src/main/java/com/gkebjb/gkebanjarbaru/MainActivity.kt
package com.gkebjb.gkebanjarbaru

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import com.gkebjb.gkebanjarbaru.databinding.ActivityMainBinding
import com.gkebjb.gkebanjarbaru.home.HomeScreenActivity

class MainActivity : AppCompatActivity() {
    private lateinit var mainBinding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        mainBinding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(mainBinding.root)

        Handler(Looper.myLooper()!!).postDelayed({
            val intent = Intent(this, HomeScreenActivity::class.java)
            startActivity(intent)
            finish()
        }, 3000)
    }
}